xmonad-contrib.cabal

```diff
@@ -23,7 +23,7 @@ license:            BSD3
 license-file:       LICENSE
 author:             Spencer Janssen & others
 maintainer:         xmonad@haskell.org
-extra-source-files: README.md CHANGES.md scripts/generate-configs scripts/run-xmonad.sh
+extra-source-files: README.md CHANGES.md NIX.md scripts/generate-configs scripts/run-xmonad.sh
```

Comment:
We aren't shipping the nix flake in the tarball, so does it make sense to ship a doc for it?

Reply:
Hm, you're probably right.  I'll push a commit to remove that at some point later today